rails-snow-queen/app/views/quote_mailer/new_quote.html.erb

```diff
@@ -0,0 +1,20 @@
+<h1>Snow Queen service request #<%= @quote.id %></h1>
+
+<h2>Hi <%= @quote.first_name %></h1>
+
+<p>
+  Thank you for choosing Snow Queen to fulfill your winter snow clearing and salting needs! <br>
+  For your reference, here are the details of the service you requested on <%= @quote.created_at %>:
```

Comment:
Might want to format the time, otherwise it comes out pretty geeky looking (see [`DateTime#strftime`](https://ruby-doc.org/stdlib-2.3.1/libdoc/date/rdoc/DateTime.html#method-i-strftime) or [`DateTime#to_formatted_s`](https://api.rubyonrails.org/classes/DateTime.html#method-i-to_formatted_s))